src/Management/src/Abstractions/PublicAPI.Unshipped.txt

```diff
@@ -9,42 +8,6 @@ Steeltoe.Management.Configuration.EndpointPermissions
 Steeltoe.Management.Configuration.EndpointPermissions.Full = 2 -> Steeltoe.Management.Configuration.EndpointPermissions
 Steeltoe.Management.Configuration.EndpointPermissions.None = 0 -> Steeltoe.Management.Configuration.EndpointPermissions
 Steeltoe.Management.Configuration.EndpointPermissions.Restricted = 1 -> Steeltoe.Management.Configuration.EndpointPermissions
-Steeltoe.Management.Diagnostics.DiagnosticObserver
-Steeltoe.Management.Diagnostics.DiagnosticObserver.DiagnosticObserver(string! name, string! listenerName, Microsoft.Extensions.Logging.ILoggerFactory! loggerFactory) -> void
-Steeltoe.Management.Diagnostics.DiagnosticObserver.Dispose() -> void
-Steeltoe.Management.Diagnostics.DiagnosticObserver.ListenerName.get -> string!
-Steeltoe.Management.Diagnostics.DiagnosticObserver.ObserverName.get -> string!
-Steeltoe.Management.Diagnostics.DiagnosticObserver.Subscribe(System.Diagnostics.DiagnosticListener! listener) -> void
-Steeltoe.Management.Diagnostics.IDiagnosticObserver
-Steeltoe.Management.Diagnostics.IDiagnosticObserver.ObserverName.get -> string!
-Steeltoe.Management.Diagnostics.IDiagnosticObserver.Subscribe(System.Diagnostics.DiagnosticListener! listener) -> void
-Steeltoe.Management.Diagnostics.IDiagnosticsManager
-Steeltoe.Management.Diagnostics.IDiagnosticsManager.Start() -> void
-Steeltoe.Management.Diagnostics.IDiagnosticsManager.Stop() -> void
-Steeltoe.Management.Diagnostics.IRuntimeDiagnosticSource
```

Comment:
I think there's an argument (though probably undocument) in there somewhere for making `IRuntimeDiagnosticSource` extensible, so maybe that should stay here

Reply:
Basically the same as `DiagnosticObserver`, though I'm a little less confident on this one. Ultimately, we need to get to the bottom of whether we can remove these entirely